compareECLFiles.cmake

```diff
@@ -317,6 +317,12 @@ add_test_compare_restarted_simulation(CASENAME spe9
                                       ABS_TOL ${abs_tol_restart}
                                       REL_TOL ${rel_tol_restart})
 
+add_test_compare_restarted_simulation(CASENAME udq_actionx
+                                      FILENAME UDQ_WCONDPROD
+                                      SIMULATOR flow
+                                      ABS_TOL ${abs_tol_restart}
+                                      REL_TOL ${rel_tol_restart})
+
```

Comment:
> The restarted case must be run with the non default option `--sched-restart=True` - it was not clear to me how to add that argument driver

Try rewriting the last line into the following two lines:
`REL_TOL ${rel_tol_restart}
TEST_ARGS --sched-restart=true)`

Reply:
Thank you - that worked; for some reason I had started my work from an ancient git ref.